tensorflow_probability/python/experimental/mcmc/sample_fold.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2020 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Drivers for streaming reductions framework."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# Dependency imports
+import tensorflow.compat.v2 as tf
+from tensorflow_probability.python.experimental.mcmc import with_reductions
```

Comment:
also a curious question here. No matter what I try it seems I can't import `WithReductions` (i.e. `from tensorflow_probability.python.experimental.mcmc import WithReductions`)

Reply:
Don't.  Import mcmc and refer to the symbol as `mcmc.WithReductions`.